ynput/ayon-core
# Loader tools: Fix sorting of versions in comboboxes

## Changelog Description
Hero versions are not at correct place in versions combobox.

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

## Screenshots
Before this PR 
![image](https://github.com/ynput/ayon-core/assets/43494761/78508252-bc6f-4767-bef8-9168e01fc053)

With this PR
![image](https://github.com/ynput/ayon-core/assets/43494761/11b17dd5-1346-4dab-aa54-268c14e5486f)


## Testing notes:
1. Open loader tool (from tray or in host).
2. Try to look at versions ordering on product which has hero version.
3. The hero version should be after it's standard version.
